### PR TITLE
21137 - validation messages and styles

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:21137*https://github.com/bcgov/entity/issues/21137

*Description of changes:*
- fixed validation messages mentioned in the ticket 21137
- fixed style issues mentioned in 21137

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-170-fy0bwih9.web.app

**Review by kialj876 — APPROVED** [review]
Nice work Hrvoje, looks good. I think there's a commented out test for some of the validation messages? Also have a comment below if you agree

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-170-fy0bwih9.web.app

File: btr-web/btr-common-components/components/bcros/inputs/CountriesOfCitizenship/Index.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <UFormGroup
+    v-slot="{ error }"
```
[the remark this answers is not in the document]
Comment (hfekete) [reply btr-web/btr-common-components/components/bcros/inputs/CountriesOfCitizenship/Index.vue]: I very rarely have preference for readability, on short stuff like that so I will agree with you :D

**Pull request merged**